js/ui/search.js

```diff
@@ -333,7 +333,7 @@ const ListSearchResults = new Lang.Class({
     },
 
     _createResultDisplay: function(meta) {
-        return this.parent(meta) || new ListSearchResult(this.provider, meta);
```

Comment:
I don't believe there's anything that would override this for list results, so you could remove this change